src/main/java/org/apache/commons/dbcp2/BasicDataSourceMXBean.java

```diff
@@ -201,13 +201,6 @@ default String getDefaultSchema() {
      */
     int getNumIdle();
 
-    /**
```

Comment:
-1: You cannot break binary compatibility, as I mentioned on the mailing list already.

Reply:
@garydgregory Thanks for your fast feedback, I'll investigate if its possible to "filter" by attribute while register MBean. Since the method was public its exported. Any suggestions?